parsons/etl_helpers/conversions.py

```diff
@@ -0,0 +1,188 @@
+"""
+This module contains functions that can be passed to the Parsons Table convert_column
+method. They take in an original value and return a new value. They can, of course, also be used
+on their own.
+"""
+
+from dateutil.parser import parse
+import datetime
+import re
+
+
+# Datetime conversion functions
+
+
+def date_to_timestamp(value, tzinfo=datetime.timezone.utc):
+    """Convert any date value into a Unix timestamp.
+
+    `Args:`
+        value: int or str or datetime
+            Value to parse
+        tzinfo: datetime.timezone
+            `Optional`: Timezone for the datetime; defaults to UTC.
+    `Returns:`
+        Unix timestamp (int)
+    """
+
+    parsed_date = parse_date(value)
+
+    if not parsed_date:
+        return None
```

Comment:
Either the function is misdocumented, or this is incorrect. What if you used `-1` as a sentinel values instead of `None`?